backend/test/resources.js

```diff
@@ -0,0 +1,136 @@
+const request = require("supertest");
+const { expect } = require("chai");
+require("./auth_stubs").stubAllAuth();
+const app = require("../app.js");
+const Resource = require("../models/Resource");
+
+const sampleResourceInfo = {
+  companyName: "Google",
+  contactName: "Alice",
+  contactPhone: "123456789",
+  contactEmail: "123@gmail.com",
+  description: "Let me dream",
+  address: "Silicon Valley",
+  location: {
+    coordinates: [35, 40]
+  },
+  tags: ["tech"]
+};
+
+const sampleResourceInfo2 = {
+  companyName: "Facebook",
+  contactName: "Bob",
+  contactPhone: "123456789",
+  contactEmail: "123@gmail.com",
+  description: "Let me dream",
+  address: "Silicon Valley",
+  location: {
+    coordinates: [40, 40]
+  },
+  tags: ["social"]
+};
+
+const createSampleResource = async (resourceInfo = sampleResourceInfo) => {
+  const newResource = new Resource(resourceInfo);
+  await newResource.save();
+};
+
+const createSampleResource2 = async (resourceInfo = sampleResourceInfo2) => {
+  const newResource = new Resource(resourceInfo);
+  await newResource.save();
+};
+
+beforeEach(() => createSampleResource());
```

Comment:
Nit: What do you think about removing this and adding this to individual `describe` blocks when necessary? This way upon reaching each `describe` block, have a clean slate in our database to work with.